server/sonar-server/src/main/java/org/sonar/server/updatecenter/ws/UploadAction.java

```diff
@@ -63,7 +62,7 @@ public void define(WebService.NewController context) {
 
   @Override
   public void handle(Request request, Response response) throws Exception {
-    userSession.checkPermission(GlobalPermissions.SYSTEM_ADMIN);
```

Comment:
description should be fixed, root is now required, not 'Administer System' permission... or is that the name we give to root?